Rakefile

```diff
@@ -27,6 +29,7 @@ RuboCop::RakeTask.new(:internal_investigation)
 task default: %i[documentation_syntax_check spec ascii_spec internal_investigation]
 
 require 'yard'
+
 YARD::Rake::YardocTask.new
```

Comment:
hmm, I'm sorry to suggest this myself, but I don't seem to want to add an empty line to this kind of code, and there may be too many cases where this Cop won't work well.

This rule might have worked in a language like JavaScript, where there is an advantage to grouping `import` at the beginning of a file for static imports, but in Ruby, `require` often appears abruptly in the middle of a file, like this code.